python/src/opendp/_convert.py

```diff
@@ -327,6 +339,10 @@ def str_to_slice(val):
             return ctypes.c_char_p(val.encode())
         array = (ctypes.c_char_p * len(val))(*map(str_to_slice, val))
         return _wrap_in_slice(array, len(val))
+    
+    if inner_type_name == "SeriesDomain":
+        array = (Domain * len(val))(*val) # type: ignore[operator]
```

Comment:
`Domain * len(val)` is a callable? More explanation, in general, and about the type ignore, would be nice.